src/test/java/uk/gov/pay/api/it/CreatePaymentITest.java

```diff
@@ -0,0 +1,233 @@
+package uk.gov.pay.api.it;
+
+import com.jayway.jsonassert.JsonAssert;
+import com.jayway.restassured.response.ValidatableResponse;
+import org.junit.Test;
+import uk.gov.pay.api.model.Address;
+import uk.gov.pay.api.model.CardDetails;
+import uk.gov.pay.api.model.PaymentState;
+import uk.gov.pay.api.model.RefundSummary;
+import uk.gov.pay.api.utils.DateTimeUtils;
+import uk.gov.pay.api.utils.JsonStringBuilder;
+
+import javax.ws.rs.core.HttpHeaders;
+import java.io.InputStream;
+import java.time.ZonedDateTime;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static javax.ws.rs.core.HttpHeaders.AUTHORIZATION;
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphanumeric;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.Is.is;
+import static uk.gov.pay.api.model.TokenPaymentType.CARD;
+
+public class CreatePaymentITest extends PaymentResourceITestBase {
+
+    private static final ZonedDateTime TIMESTAMP = DateTimeUtils.toUTCZonedDateTime("2016-01-01T12:00:00Z").get();
+    private static final int AMOUNT = 9999999;
+    private static final String CHARGE_ID = "ch_ab2341da231434l";
+    private static final String CHARGE_TOKEN_ID = "token_1234567asdf";
+    private static final PaymentState CREATED = new PaymentState("created", false, null, null);
+    private static final RefundSummary REFUND_SUMMARY = new RefundSummary("pending", 100L, 50L);
+    private static final String PAYMENT_PROVIDER = "Sandbox";
+    private static final String CARD_BRAND_LABEL = "Mastercard";
+    private static final String RETURN_URL = "https://somewhere.gov.uk/rainbow/1";
```

Comment:
We should probably use a `.test` url as @alexbishop1 would suggest

Reply:
I’m actually OK with `*.something.gov.uk` domains because we’re the government and it’s unlikely to result in grief or embarrassment for anyone.

It’s other random potentially-registered domains I worry about.